DFC.ServiceTaxonomy.JobProfiles.Module/ServiceBusHandling/Converters/JobProfileMessageConverter.cs

```diff
@@ -60,10 +63,16 @@ public JobProfileMessage ConvertFrom(ContentItem contentItem)
                     HowToBecomeData = _howToBecomeMessageConverter.ConvertFrom(contentItem),
                     WhatYouWillDoData = _whatYouWillDoDataMessageConverter.ConvertFrom(contentItem),
                     RelatedCareersData = GetRelatedCareersData(relatedCareersProfiles),
-                    Restrictions = GetRestrictions(restrictions),
-                    OtherRequirements = contentItem.Content.JobProfile.Otherrequirements == null ? default(string?) : (string?)contentItem.Content.JobProfile.Otherrequirements.Html,
+                    //Restrictions = GetRestrictions(restrictions),
+                    //OtherRequirements = contentItem.Content.JobProfile.Otherrequirements == null ? default(string?) : (string?)contentItem.Content.JobProfile.Otherrequirements.Html,
                     DynamicTitlePrefix = dynamicTitlePrefix.Any() ? dynamicTitlePrefix.First().As<TitlePart>().Title : string.Empty,
-                    DigitalSkillsLevel = digitalSkillsLevel.Any() ? digitalSkillsLevel.First().Content.Digitalskills.Description.Text : string.Empty
+                    //DigitalSkillsLevel = digitalSkillsLevel.Any() ? digitalSkillsLevel.First().Content.Digitalskills.Description.Text : string.Empty,
+                   
+
+                    //SocSkillsMatrixData - TODO: RelatedSkills to be added later
+                    DigitalSkillsLevel = _whatItTakesMessageConverter.ConvertFrom(contentItem).RelatedDigitalSkills,
```

Comment:
You don't have to call the converter multiple times, can create a local variable to keep the result and use that variable to assign these properties. like
```
var witMessage = _whatItTakesMessageConverter.ConvertFrom(contentItem);
```
and then later 
```
DigitalSkillsLevel = witMessage.RelatedDigitalSkills,
// and so on
```

Reply:
done